resources/views/partials/errors/duplicate-domain.blade.php

```diff
@@ -1,3 +1,3 @@
 <li class="padding">
-	{!! sprintf(__('The email domain %s is already in use with %s. Please use a different address.', 'pressbooks-multi-institution'), $domain, $institution) !!}
+	{!! sprintf(__('The email domain %s is already in use with %s. <strong>Please use a different address.</strong>', 'pressbooks-multi-institution'), $domain, $institution) !!}
```

Comment:
Is it possible to split the message so we don't have `<strong>` tags inside of the message?
```suggestion
	{!! sprintf(__('The email domain %s is already in use with %s.', 'pressbooks-multi-institution'), $domain, $institution) !!}
	<strong>{{ __('Please use a different address.', 'pressbooks-multi-institution') }}</strong>
```